frontend/packages/kubevirt-plugin/src/statuses/template/template-source-status.ts

```diff
@@ -42,13 +49,30 @@ export const getTemplateSourceStatus: GetTemplateSourceStatus = ({
   pods,
   pvcs,
   dataVolumes,
+  dataSources,
 }) => {
   if (isCommonTemplate(template)) {
     const baseImageName = getParameterValue(template, TEMPLATE_BASE_IMAGE_NAME_PARAMETER);
     const baseImageNs = getParameterValue(template, TEMPLATE_BASE_IMAGE_NAMESPACE_PARAMETER);
     if (!baseImageName || !baseImageNs) {
       return null;
     }
+
+    const matchedDataSource = dataSources?.find(({ metadata }) => metadata?.name === baseImageName);
+    if (matchedDataSource) {
+      const findFunction = findByDataSource(matchedDataSource);
+      const matchedDataSourcePVC = pvcs?.find(findFunction);
+      const matchedDataSourceDataVolume = dataVolumes?.find(findFunction);
+      return {
+        source: SOURCE_TYPE.BASE_IMAGE,
+        provider: 'Red Hat',
```

Comment:
minor nitpick: Do we not have a constant for this anywhere?

Reply:
its funny but across all kubevirt we didn't had Red Hat const, so I added one now. thank you!